commands/russianarmylosses/index.js

```diff
@@ -0,0 +1,91 @@
+module.exports = {
+	Name: "russianarmylosses",
+	Aliases: ["ral"],
+	Author: "boring_nick",
+	Cooldown: 15000,
+	Description: "Fetches the latest losses of the Russian Army in Ukraine as provided by the General Staff of Ukraine.",
+	Flags: ["mention", "non-nullable", "pipe"],
+	Params: [
+		{ name: "term", type: "string" }
+	],
+	Whitelist_Response: null,
+	Static_Data: (() => ({
+		// While this data is provided by the API (https://russianwarship.rip/api-documentation/v1#/Terms/getAllStatisticalTerms),
+		// some of the names are too long for chat and have been shortened.
+		terms: {
+			personnel_units: "Personnel",
+			tanks: "Tanks",
+			armoured_fighting_vehicles: "Combat vehicles",
+			artillery_systems: "Artillery",
+			mlrs: "MLRS",
+			aa_warfare_systems: "AA",
+			planes: "Planes",
+			helicopters: "Helicopters",
+			vehicles_fuel_tanks: "Vehicles/Fuel tanks",
+			warships_cutters: "Ships",
+			cruise_missiles: "Missiles",
+			uav_systems: "UAV",
+			special_military_equip: "Special equipment",
+			atgm_srbm_systems: "ATGM/SRBM"
+		}
+	})),
+	Code: (async function russianArmyLosses (context) {
+		const terms = this.staticData.terms;
+		const { term } = context.params;
```

Comment:
This is more of a suggestion to think about rather than a request change, but I think using the first word from the user input is also valid - instead of having to always rely on a parameter. Using parameters is always a little more difficult from UX standpoint, and I don't really see a reason to not support direct arguments here.